utils/consts.go

```diff
@@ -31,6 +31,7 @@ const (
 	jfrogReleasesRepoEnv   = "JF_RELEASES_REPO"
 	JFrogPasswordEnv       = "JF_PASSWORD"
 	JFrogTokenEnv          = "JF_ACCESS_TOKEN"
+	JfrogConfigProfileEnv  = "JF_CONFIG_PROFILE" // JFROG internal use ONLY!
```

Comment:
I'd remove the comment. It won't be in the official documentation until it's GA for all.

Reply:
Also comments should be above the line and inline should be avoided